### PR TITLE
[PWD-2524] Finalize Event Page

### PR DESCRIPTION
**Pull request by kuntiarso**
### What type of PR is this? (_check all applicable_)

- [x] Feature
- [ ] Bug Fix
- [ ] Documentation Update
- [ ] Setup
- [ ] Code Refactor
- [ ] Performance Improvements
- [ ] Test
- [ ] Build
- [ ] CI
- [ ] Revert

### Screenshots (_optional_)

emtpy event
Screenshot 2023-08-04 093825

standard lg screen
Screenshot 2023-08-04 093903

xl screen
Screenshot 2023-08-04 093919

### CODE REVIEW

File: src/views/event/components/EventCard.tsx [hunk 1]
```diff
@@ -37,15 +37,14 @@ const EventCard: FC<EventCardProps> = ({ data, daftarEventHandler }) => {
...
-      height="653px"
       bgcolor="neutral900.main"
       sx={{ borderRadius: '6px' }}
     >
       <img
         src={baseUrl.concat(data.image)}
         width="100%"
         height="157px"
-        style={{ objectFit: 'cover' }}
+        style={{ objectFit: 'cover', borderRadius: '6px 6px 0px 0px' }}
```
Comment (hiljaker) [inline src/views/event/components/EventCard.tsx:47]: gimana kalau 
```suggestion
        style={{ objectFit: 'cover' }}
```
dan stacknya dikasih overflow hidden

Comment (kuntiarso) [reply src/views/event/components/EventCard.tsx:47]: ih keren bgt lu

**Review by hiljaker — APPROVED** [review]
jumat mubarak 🕌✨

**Review by Vesals — APPROVED** [review]
sbelum jumat'an

**Pull request merged**